src/test/tst_value.cpp

```diff
@@ -89,44 +70,148 @@ void TestValue::tst_compare()
     QVERIFY_EXCEPTION_THROWN(v1 <= v2, std::invalid_argument);
     QVERIFY_EXCEPTION_THROWN(v1 > v2, std::invalid_argument);
     QVERIFY_EXCEPTION_THROWN(v1 < v2, std::invalid_argument);
+}
+
+void TestValue::tst_valueBool(){
+    Value v;
+
+    //VALUE TESTS
+    //case where v is set to true
+    v = Value(true);
+
+    //Testing validity
+    QVERIFY(v.isValid() == true);
 
-    // compare bool
+    //Testing type
+    QVERIFY(v.type() == Value::BOOL);
+
+    //Testing value
+    QVERIFY(v.toBool() == true);
+    QCOMPARE(v.toQString(), QString("1"));
+
+    //case where v is set to false
+    v = Value(false);
+
+    //Testing validity
+    QVERIFY(v.isValid() == true);
+
+    //Testing type
+    QVERIFY(v.type() == Value::BOOL);
+
+    //Testing value
+    QVERIFY(v.toBool() == false);
```

Comment:
just for consistency, as we're retrieving the value, it's better to use QCOMPARE
``` c
QCOMPARE(v.toBool(), false);
```